1_fetch.R

```diff
@@ -118,6 +131,41 @@ p1_targets_list <- list(
   tar_target(
     p1_reaches_sf,
     st_read(p1_reaches_shp)
+  ),
+
+  # fetch prms met data
+  tar_target(
+    p1_prms_met_data_zip_file,
+    download_sb_file(sb_id = "5f6a289982ce38aaa2449135",
+                     file_name = "sntemp_inputs_outputs_drb.zip",
+                     out_dir = "1_fetch/out"),
+    format = "file"
+  ),
+
+  # unzip prms met data
+  tar_target(
+    p1_prms_met_data_csv_file,
+    {
+    unzip(zipfile=p1_prms_met_data_zip_file,exdir = dirname(p1_prms_met_data_zip_file),overwrite=TRUE)
+    file.path(dirname(p1_prms_met_data_zip_file), "sntemp_inputs_outputs_drb.csv")
+    },
+    format = "file"
+  ),
+
+  # read in prms met data
+  tar_target(
+    p1_prms_met_data,
+    read_csv(p1_prms_met_data_csv_file)
+  ),
+
+  # read in prms met data
+  # [Jeff] I'm including these in the "in" folder because they are unpublished
+  # They are built in the delaware_model_prep pipeline (1_network/out/seg_attr_drb.feather)
```

Comment:
This makes good sense to me. Do you know what the reason was for using .feather files in `delaware-model-prep`? Faster read/write? Can .feather files be read into python, too?

Reply:
Good question. I was the one that wrote all of this and now looking back, I don't think it was a good idea. I think csv would be much better. You can read feather in Python. I did it in feather b/c it is slightly smaller and because it preserves the data types, but now I think it would be better just to do csv for simplicity/transparency. That said, I'm not sure if/when it's worth switching that over.

Reply:
Thanks for the context! R has a file format (.rds) that preserves data types and I think has faster read/write, but I'm hesitant to use it sometimes because it doesn't seem portable to other languages - it's useful that feather preserves data types and can be read into Python too ❤️